docs/guides/kubernetes.md

```diff
@@ -202,7 +147,9 @@ The policy should be a base64-encoded block of yaml:
   kubernetes_service_account_token: "..." #$(kubectl get secret/"$(kubectl get serviceaccount/pomerium -o json | jq -r '.secrets[0].name')" -o json | jq -r .data.token | base64 -d)
 ```
 
-Applying this configuration will create a Pomerium deployment and service within kubernetes that is accessible from `*.localhost.pomerium.io:30443`.
+:::
+
+Applying this configuration change will create a Pomerium route within kubernetes that is accessible from `*.localhost.pomerium.io:30443`.
 
 ## Kubectl
 
```

Comment:
@travisgroth the section below says it's installing something kubernetes-specific, but it's really the pomerium-cli, right? Should I rewrite it to name it as such, link to our docs on it, and reference our existing install instructions for it?

Reply:
It's installing `pomerium-cli` - which we should update the instructions for - but then configuring `kubectl` to use it as an auth helper.  Maybe rename to

  ```
## Configure kubectl
```

And link to install instructions for the cli.